src/commands/hardis/org/monitor/all.ts

```diff
@@ -80,6 +80,7 @@ monitoringCommands:
       { title: "Detect suspect setup actions in major org", command: "sfdx hardis:org:diagnose:audittrail" },
       { title: "Detect calls to deprecated API versions", command: "sfdx hardis:org:diagnose:legacyapi" },
       { title: "Detect custom elements with no access rights defined in permission sets", command: "sfdx hardis:lint:access" },
+      { title: "Detect custom labels that are not in use", command: "sfdx hardis:lint:unusedMetadatas" },
```

Comment:
Please change the label as you detect multiple unused metadatas, not just the label ^^